partials/forms/radio-group.html

```diff
@@ -1,4 +1,5 @@
-<div id="{{key}}-group" class="form-group{{#display}} {{display}}{{/display}}{{#error}} validation-error{{/error}}">
+<fieldset id="{{key}}-group" class="{{#display}}{{display}}{{/display}}{{#error}} validation-error{{/error}}">
+    <legend{{#legend-display}} class="{{legend-display}}"{{/legend-display}}>{{legend}}</legend>
```

Comment:
If we only have one value at the moment then I'd just use that value, and we can make it configurable if/when it's needed.